ethcore/src/state/mod.rs

```diff
@@ -95,6 +97,7 @@ enum AccountState {
 /// account (`None`)
 struct AccountEntry {
 	account: Option<Account>,
+	clean_balance: Option<U256>,
```

Comment:
probably worth documenting these members now - `clean_balance` isn't immediately obvious in meaning. would probably rename to `old_balance` or something similar to emphasise its underlying semantics.